.pre-commit-config.yaml

```diff
@@ -1,15 +1,31 @@
 repos:
-  - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
     hooks:
-      - id: isort
-        name: isort (python)
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
   - repo: https://github.com/kynan/nbstripout
-    rev: 0.5.0
+    rev: 0.6.0
     hooks:
       - id: nbstripout
         args: [--extra-keys="cell.metadata.pycharm" --strip-empty-cells]
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args:
+          [
+            "--profile",
+            "black",
+            "--filter-files",
+            "--skip",
+            "tests/neptune/test_imports.py",
```

Comment:
I guess we don't need `"--filter-files", "--skip", "tests/neptune/test_imports.py",` here?

Reply:
we don't. I've let it be just in case we add tests in the future.
doesn't impact code.

Reply:
removed this to make the config cleaner